iOS/Services/StitchAWSS3Service/StitchAWSS3ServiceTests/AWSS3ServiceClientIntTests.swift

```diff
@@ -50,7 +50,7 @@ class AWSS3ServiceClientIntTests: BaseStitchIntTestCocoaTouch {
         }
         wait(for: [exp0], timeout: 5.0)
         
-        let awsS3 = client.serviceClient(forFactory: AWSS3Service.sharedFactory, withName: "aws-s31")
+        let awsS3 = client.serviceClient(forFactory: awsS3ServiceClientFactory, withName: "aws-s31")
```

Comment:
[minor] I know this wasn't introduced here, but this param name is wrong. you get a service client using a factory, not for a factory. is there a better name?

Reply:
`serviceClient(`usingFactory`: ...)`

Reply:
`fromFactory`?

Reply:
Sure